lib/appmap/service/guesser.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module AppMap
+  module Service
+    class Guesser
+      POSSIBLE_PATHS = %w[app/controllers app/models lib]
+      class << self
+        def guess_name
+          reponame = lambda do
+            next unless File.directory?('.git')
+
+            repo_name = `git config --get remote.origin.url`.strip
+            repo_name.split('/').last.split('.').first unless repo_name == ''
+          end
```

Comment:
There are a couple of problems with this way of guessing the repo name: the default remote won't necessarily be  named `origin`, and the repo name might have a `.` in it (e.g. https://github.com/chicagopython/chipy.org).

I think would be better to just use the name of the root directory of the repo, i.e. `basename $(git rev-parse --show-toplevel)`.